extensions/packageManager/src/client/Library.tsx

```diff
@@ -17,7 +17,7 @@ import { WorkingModal } from './workingModal';
 
 const DEFAULT_CATEGORY = formatMessage('Available');
 
-const docsUrl = `https://aka.ms/composer-use-package-library`;
+const docsUrl = `https://github.com/microsoft/BotFramework-Composer/blob/main/extensions/packageManager/readme.md`;
```

Comment:
Why this change?  Can we just redirect the aka.ms link instead?

Reply:
Sure. I updated this link to: 
https://aka.ms/composer-package-manager-readme